Changelog

```diff
@@ -1,9 +1,31 @@
 Changelog for Percona Toolkit
 
- * Improvement PT-1974: Support fingerprinting for --print in pt-kill (Thanks Iwo Panowicz)
- * Fixed bug   PT-1966: Fixed test (Thanks @yoku0825)
+v3.3.2 WIP release date not set yet
+
+ * Improvement PT-1417: Inconsistent creation of toolkit tables
+ * Fixed bug   PT-1627:	pt-mysql-summary doesn't verify which version of jemalloc is in use
+ * Fixed bug   PT-1747: pt-online-schema-change: metadata lock can break database for rebuild_constraints
+ * Improvement PT-1800: PTDEBUG=1 exposes passwords
+ * Fixed bug   PT-1900:	pt-query-digest not hiding parameter properly sometimes when parameter=binary
+ * Improvement PT-1940:	ptsoc dropswap with mysql8: revise rejection (Thanks duxthefux)
+ * Fixed bug   PT-1953: pt-summary typo: Memory mamagement.
+ * Fixed bug   PT-1959: go part of the toolkit still has the version 3.3.0
  * Fixed bug   PT-1965: pt-stalk --mysql-only doesn't collect mysqladmin outputs (Thanks Sergey Kuzmichev)
+ * Fixed bug   PT-1966: Test no_drop_no_swap for the pt-online-schema-change is broken (Thanks Tsubasa Tanaka)
+ * Fixed bug   PT-1974: Support fingerprinting for --print in pt-kill
+ * Fixed bug   PT-1983:	pt-summary missing one DIMM	
+ * Admin task  PT-2009: Update Go Lang dependencies
+ * Admin task  PT-2011:	Make build reproducible
```

Comment:
This is an Admin task. We usually do not include them in RN. 
 They need to be removed or converted to another appropriate type if this  a bug or improvement.

Reply:
do we do RNs from Changelog? this is a changelog....

Reply:
Yes. This is only the changelog to track what we are going to include in next release.